docs/src/index.rst

```diff
@@ -246,14 +303,105 @@ Usage example
                 return 0;
             }
 
+    .. tab-item:: TorchScript Python
+
+        The entry point for the TorchScript API is the
+        :py:class:`vesin.torch.NeighborList` class in Python, and the
+        corresponding :cpp:class:`vesin_torch::NeighborListHolder` class in C++;
+        both modeled after the standard Python API. For Python, the class is
+        available in the ``vesin.torch`` module.
+
+        In both cases, the code is integrated with PyTorch autograd framework,
+        meaning if the ``points`` or ``box`` argument have
+        ``requires_grad=True``, then the ``d`` (distances) and ``D`` (distance
+        vectors) outputs will be integrated to the computational graph.
+
+        .. code-block:: Python
+
+            import torch
+            from vesin.torch import NeighborList
+
+            positions = torch.tensor(
+                [[0.0, 0.0, 0.0],
+                 [0.0, 1.3, 1.3]],
+                dtype=torch.float64
+            )
+            box = 3.2 * torch.eye(3, dtype=torch.float64)
```

Comment:
Do you want to set

```python
positions.requires_grad = True
box.requires_grad = True
```

 before you call `calculator.compute` to show when one has to enable the gradients?

Reply:
isn't the small text above enough for this?

Reply:
might be stupid, but users may put the `positions.requires_grad = True` below the actual computation and wonder why it is not correct.